### ISSUE TITLE
Update roadmap & more clearly define what Open Drupal does

### ISSUE THREAD

**Issue by hedleysmith** [OpenDrupal/opendrupal#27]
After a very interesting chat with Joeri Poesen about what Open Drupal is and where it's going I think it's a good time to rethink these things. We both thought that Open Drupal needs clearer aims and that open training materials will be beneficial in many contexts but that there should be more room for adaptation based upon an agreed standard.

So, the big picture is that right now I think it makes sense to focus on the curriculum as a complete training package for the UK Drupal apprenticeship qualification, so there's a clear achievable target to guide development.

Then when 'version 1' is complete immediately move on to the next version which in which we will 'decouple' the curriculum from the training material, potentially allowing an open standard which other's can use as the basis for their own training material.

I think it will be easier to do it this way around because 1) there is time pressure to complete something for the UK Drupal apprenticeship programme 2) I think it will be easier to flesh out the aims & learning objectives for the curriculum after writing the training materials as we'll have a proven 'implementation' already.

__Proposed new aims of Open Drupal__

*Main aim:* Free and open source training material by way of The Open Drupal Curriculum.
*Secondary aims:* Support local training programmes and provide tools to help training and learning Drupal.

__Proposed immediate goals (2015?):__

1. Complete version 1 of the curriculum (using markdown files in Github), which will be used for the UK Drupal Apprenticeship qualification and any other training that people want to use it for.
2. Restructure opendrupal.org, see [this issue](https://github.com/OpenDrupal/opendrupal/issues/26).

__Proposed future goals (2015/2016?):__

* Build a frontend for the curriculum, to display the markdown files in a nicer way (probably using a static-site generator).

Version 2 of the curriculum:
* Include training for Drupal 8
* Decouple the curriculum from the training materials by writing more comprehensive aims & learning objectives.
* Clearer instructions on implementing and contributing to The Open Drupal Curriculum.

After we've decided we can incorporate into the [roadmap.md](https://github.com/OpenDrupal/opendrupal/blob/master/roadmap.md) file.

**Comment by mgifford** [comment]
This is great to see. Would be nice if this could integrate with Mozilla's Open Badges & also be incorporated into Drupal.org in general.  I've posted some material about this here:
https://www.drupal.org/node/2203401

**Comment by hedleysmith** [comment]
Yeah, I'm really keen for Open Badges integration. I haven't yet used them but I've read up and they seem perfect for the curriculum. It would be really awesome if a trainer could award an Open Badge after completing a unit! I'll add this to the roadmap.

On your note about drupal.org integration - do you mean integrating opendrupal.org with drupal.org, or integrating Open Badges with drupal.org?

**Comment by mgifford** [comment]
Ideally I think that the opendrupal.org initiative could fold into a broader recognition program that was supported by Drupal.org.  

So, yes..  I mean both integrating Open Badges with Drupal.org & OpenDrupal.

**Comment by hedleysmith** [comment]
Yeah folding Open Drupal into drupal.org or have a supporting connection between the two is something I've thought might be a natural evolution (I've always thought a learn.drupal.org site could be really beneficial) - I think this might be best a longer term thing though that's discussed when the concept has been 'proven' on it's own. Definitely not against this sooner if anyone wants to pursue it though :)

**Comment by crispinread** [comment]
@hedleysmith  I think you have this spot on - Tech Partnership John said we don't have to have everything complete before we start so how about we create milestones based on the progress of the next intake? Set appropriate deadlines for draft review and edits so the material is ready in good time.

@mgifford I'd love to see Open Badges integration too

**Comment by hedleysmith** [comment]
Good call @crispinread I'll get all the workshops we have planned in [here](https://github.com/OpenDrupal/opendrupal/tree/master/workshops) and we can work backwards form there to see which units we'll need to complete first. Maybe it could be as simple as three milestones, one for completion ASAP, one for 6 months in and one for everything else...

**Comment by hedleysmith** [comment]
OK, I've committed these updated goals to the [roadmap](https://github.com/OpenDrupal/opendrupal/blob/master/roadmap.md)
